src/templates/rest_framework/api.html

```diff
@@ -0,0 +1,4 @@
+<html lang=nl>
```

Comment:
Dit genereert foutieve HTML:
```

  | <html lang=nl>
-- | --
  |  
  |  
  |  
  |  
  | <!DOCTYPE html>
  | <html>
  | <head>
  |  


```

Taal moet of via `<meta>` of via Headers gezet worden.
Zie: https://github.com/encode/django-rest-framework/blob/master/rest_framework/templates/rest_framework/base.html#L10

of zet taal via settings `LANGUAGE_code`

Reply:
Dank je voor review! Ik was vergeten te vermelden dat ik de lang gekozen conform de W3 specs. https://www.w3.org/International/questions/qa-html-language-declarations Ze raden af om de meta te gebruiken. De correcte wijze zou een lang attribuut zijn binnen de HTML tag. Geeft dit bij jou een foutieve HTML terug?